source/slang/slang-modifier-defs.h

```diff
@@ -28,16 +28,26 @@ SIMPLE_MODIFIER(GloballyCoherent)
 #undef SIMPLE_MODIFIER
 
 // A modifier that marks something as an operation that
+// translates to special
```

Comment:
special...?

Reply:
Garbage text due to copy-pasting that I meant to remove. Will clean up.